src/patterns/components/message/overview.hbs

```diff
@@ -0,0 +1,10 @@
+---
+order: 1
+notes: |
+  Message pattern
+---
+
+<div class="Message">
+ Hi, I'm a message!
```

Comment:
Should this be a paragraph semantically?

Also, I'd recommend using our typical pattern structure to extend this for `warning`. For example:

```hbs
<div class="Message {{messageClass}}">
  {{#block "content"}}
    <p>Hi, I'm a message!</p>
  {{/block}}
  {{#if includeClose}}
    <button class="Message-close">
      {{#svg "icon/x" aria-labelledby=(defaultTo closeIconTitle "Message-close-title")}}
        <title id="Message-close-title">Close</title>
      {{/svg}}
    </button>
  {{/if}}
</div>
```

See my comment on the warning template for how this would make future maintenance a bit easier.